contrib/install/openshift/perceptor.yaml

```diff
@@ -28,12 +28,13 @@ items:
               - ./perceptor
             ports:
               - containerPort: 3001
+                protocol: TCP
             resources:
               requests:
                 memory: 2Gi
-                cpu: 50m
+                cpu: 100m
```

Comment:
Just curious, was perceptor not being allocated enough processor?  => would be great if this fixed the "connection refused" errors ...